sunpy/instr/rhessi.py

```diff
@@ -476,14 +480,6 @@ def backprojection(calibrated_event_list, pixel_size=(1., 1.) * u.arcsec,
     if not (isinstance(image_dim, u.Quantity) and image_dim.unit == 'pix'):
         raise ValueError("Must be astropy Quantity in pixels")
 
-    try:
```

Comment:
@Cadair : I wonder whether taking this out triggered the docs fail?

 It's definitely coming in from this PR, and I think sunpy needs to be more strict about not merging in PRs that are failing.